db_R_tutorial.Rmd

```diff
@@ -15,15 +15,59 @@ output:
 ---
 
 ```{r setup, warning=FALSE, message=FALSE}
+###################### set-up environment section ################################
+# Set the project path to the root level -
+root.dir = rprojroot::find_rstudio_root_file()
+
 knitr::opts_chunk$set(echo = TRUE)
 
-library(tidyverse)
-library(readxl)
-library(ggthemes)
-library(kableExtra)
-library(tibble)
-library(DBI)
-#library(RSQLite)
+# load tidyverse for data science such as data handling and visualization
```

Comment:
Good ideas!
The comments on the packages are definitely helpful for beginners!

The automatic installation too. Maybe we find a way to make this more compact (without repeating the same code) and easier to read. 
https://statisticsglobe.com/r-install-missing-packages-automatically
https://vbaliga.github.io/verify-that-r-packages-are-installed-and-loaded/

The downside of those approaches is that it is not so easy to understand for a beginner. 
Maybe your way is the easiest to understand (we could add a comment about what is happening there to avoid confusion for the reader).

Reply:
Yes, I think so too as I'm not a high level skill programmer in R so I often choose the more obvious way 👍